Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -260,23 +260,32 @@ In order to address these attack vectors, the outcome of the following checks sh
 
 #### Overview
 
--- TODO [Provide a general description of the issue "Testing WebView Protocol Handlers".] --
+Protocol handler is a basic form of IPC in iOS system. Basically, it allows for invoking arbitrary applications, by calling a URL scheme with specified parameters. There are some default handlers<sup>[1]</sup>, however a developer is allowed to register his own custom URL scheme<sup>[2]</sup>. Unfortunately, this brings a serious security concern, what found out developers of Skype application<sup>[3]</sup>. The Skype application registered `skype://` protocol handler, which allows for making a call to arbitrary number without asking for a user's permission. Attackers exploited this vulnerability by putting an invisible `<iframe src=”skype://xxx?call"></iframe>` (where `xxx` was replaced by a premium number), so any Skype's user who visited malicious website unconsciously was forced to call to a premium number.
```

Comment:
You are right Pawel, that's confusing. So we have three test cases that all go somehow in the same direction:

1. Testing Custom URL Schemes
2. Testing WebView Protocol Handlers
3. Testing for Local File Inclusion in WebViews

I would say we merge 2 and 3 as it doesn't make much sense to split them. It also would make sense to merge the corresponding MASVS requirements. Testing Custom URL schemes still makes sense to have as a test case. 

MASVS requirements that should be merged:
- V6.6: "WebViews are configured to allow only the minimum set of protocol handlers required (ideally, only https is supported). Potentially dangerous handlers, such as file, tel and app-id, are disabled."
- V6.7: "The app does not load user-supplied local resources into WebViews."

@b-mueller Are you ok merging 6.6 and 6.7 and therefore also the test cases in the MSTG?